.github/workflows/ci.yml

```
@@ -85,32 +92,132 @@ jobs:
-        rm -rf src/plugins/generated/
-        mkdir -p src/plugins/generated/
-        echo "✅ Cleanup completed"
+        # 新しい品質監査スクリプトを実行
+        if [ -f scripts/quality-audit.js ]; then
+          node scripts/quality-audit.js
+        else
+          # 暫定的に既存の品質チェックを実行
+          mkdir -p .rimor/reports/quality
+          npm run analyze:src:json > .rimor/reports/quality/quality-audit-results.json
+          ANALYSIS_EXIT_CODE=$?
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)